<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrimAroundSpecificValueOptionWidget</class>
 <widget class="QWidget" name="TrimAroundSpecificValueOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>14</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Field applied to</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="fieldComboBox">
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Const field</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Preutterance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Overlap</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Target value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="targetValueSpinBox">
     <property name="minimum">
      <double>-99999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rouding range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="roundingRangeSpinBox">
     <property name="prefix">
      <string>Â±</string>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
